packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -42,7 +43,20 @@ const metadata: Meta<TableArgs> = {
             }
         </style>
     `),
-    args: {}
+    args: {
+        data: [
```

Comment:
Instead of having data as an object arg parameter, can we make some example data sets and use those instead. Similar to the drawer having content options of 'simple' vs 'complex' content. I think it would be more useful to have pre-built interesting data sets then to give this much flexibility (particularly since the editor essentially makes you hand write json)

Reply:
I'm not against that idea, and, in fact, I had a similar vision to how this file might evolve. I'm just not sure what the difference between simple & complex would be now since we aren't doing anything interesting with the data yet. Did you have something specific in mind that you'd like to see changed now, or are you ok with leaving it as-is for now and adding more interesting examples/data in the future?

Reply:
I'd say we can go ahead and organize it that way to align for future devs. Even if it is a radio with a single value of 'simple data' for now

Reply:
That seems reasonable to me. I've updated the story.